lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -225,21 +225,21 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
                         ...bucket,
                         checked: !!selectedBucket,
                         display: this.translationService.instant(bucket.display),
-                        label: this.translationService.instant(bucket.label)
+                        label: bucket.label
                     };
                 });
                 const bucketList = new SearchFilterList<FacetFieldBucket>(buckets, field.pageSize);
                 bucketList.filter = (bucket: FacetFieldBucket): boolean => {
                     if (bucket && bucketList.filterText) {
                         const pattern = (bucketList.filterText || '').toLowerCase();
-                        const label = (bucket.display || bucket.label || '').toLowerCase();
+                        const label = (bucket.display || this.translationService.instant(bucket.label || '')).toLowerCase();
```

Comment:
won't work correctly if 'display' value is set, you are translating only "label" here.